### PR TITLE
Fix for #2310

### PR DESCRIPTION
**Pull request by jokoho48**
Fix for #2310

### CODE REVIEW

File: addons/optionsmenu/config.cpp [hunk 1]
```diff
@@ -32,3 +32,10 @@ class CfgAddons {
 class ACE_Extensions {
     extensions[] += {"ace_clipboard"};
 };
+
+
+class CfgCommands {
+    allowedHTMLLoadURIs[] = {
```
Comment (jonpas) [inline addons/optionsmenu/config.cpp]: Shouldn't it be `+=` (if that works) so `allowedHTMLLoadURIs` from anything else wouldn't get deleted?

Comment (commy2) [inline addons/optionsmenu/config.cpp]: This doesn't influence the one in description.ext at all.
+= could actually be used here though. Would be a good idea. That way other addons could use it too.

Comment (jokoho48) [inline addons/optionsmenu/config.cpp]: ok i change it

**Comment by ViperMaul** [comment]
Does this also work for Arma v1.50 as well as v1.52?

**Comment by jokoho48** [comment]
@ViperMaul yes

**Comment by Glowbal** [comment]
#2428

**Pull request closed without merging**